tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
  <section title="Controller Plane (Management and Control) Considerations" anchor="cp_considerations">
 
    <t>
-	[Editor's note: This section covers management/control plane related
-	implications of creation, mapping, removal of DetNet flow IDs, their
-	related parameters and, when needed, the configuration of PREOF.]
+        TSN Stream(s) to DetNet flow mapping related information are
+        required only for the service proxy function of MPLS (DetNet) Edge nodes. 
+        From the Data Plane perspective TSN flows are application flows for 
+        DetNet. The proxy behaves as an Application interface for DetNet and 
+        as a TSN relay node for TSN Ethernet flows.         

```

Comment:
See comment above. Whole DetNet domain = TSN relay node, Proxy = port of TSN relay node.